ext/styledmap/eu.esdihumboldt.hale.doc.user.views.styledmap/html/customtile_detail.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Map configuration via Custom Tile URL</title>
+</head>
+<body>
+	<h1>Custom Tile Server Configuration</h1>
+	<p>
+		The <i>Custom Tile Maps</i> provides functionality to add map that uses web tiling scheme.
+	</p>
+	<p>
+		Here is the explanation of how to add, configure and remove <i>Custom Tile</i> 
+		map in <i>Hale</i> studio. To work with <i>Custom Tile</i> maps, first open Map perspective 
+		using Open Perspective tool menu reside in the right corner of the studio.
+    </p>
+    
+    <div>
+		<img src="../images/map_perspective_toolbar.png"
+			title="Map perspective tool menu" class="fit">
+		<img src="../images/map_perspective_open.png"
+			title="Map perspective tool menu" class="fit" style="margin-left:20px" >
+	</div>
+    
+    <p>
+		Custom tile map is a default map in <i>HALE</i>. By default you will see,
+		<i>Stamen Terrain</i> map [Map tiles by Stamen Design, under CC BY 3.0. 
+		Data by OpenStreetMap, under CC BY SA.]
+    </p>
+    
+    <div>
+		<img src="../images/custom_default.png"
+			title="Custom Tile map(Stamen Terrain) as a default in HALE" class="fit">
+		<img src="../images/custom_default_menu.png"
+			title="Stamen Terrain in view menu of map" class="fit" style="margin-left:20px" >
+	</div>
+ 
+    
+    <p>
+    	To add the <i>Custom Tile</i> maps in to Map perspective, click on the <i>View Menu</i> 
+    	button and then heads to <i>Custom Tile Maps</i>&#8594;<i>Add</i>. 
+	</p>
+	
+	<div>
+		<img src="../images/custom_add.png"
+			title="Add Custom Tile map to Map perspective" class="fit">
+	</div>
+	
+	
+	<p>
+		By clicking on <i>Add</i> button, a dialog will be appeared, which asks 
```

Comment:
Replace these two sentences with:

Click on the <i>Add</i> button, then provide a name for the service and enter the URL pattern.